source/ed/data.cpp

```diff
@@ -456,11 +468,12 @@ void Data::clean() {
     }
     vehicle_journeys.resize(num_elements);
 
-    LOG4CPLUS_INFO(logger, "Data::clean(): " << erase_overlap <<  " vehicle_journeys have been deleted because they overlap, "
-                   << erase_emptiness << " because they do not contain any clean stop_times, "
-                   << erase_no_circulation << " because they are never valid "
-                   << " and " << erase_invalid_stoptimes << " because the stop times were negatives");
-
+    if (erase_overlap){
```

Comment:
it's nice to add a condition around the log, but shouldn't it be on `erase_overlap || erase_emptiness || erase_no_circulation || erase_invalid_stoptimes` ?

The best would be to split this log :santa: